src/mtx3/utils/CommandBinaryBuffer.ts

```diff
@@ -1505,21 +1495,21 @@ export interface IOperatorParametersExtended2 {
      *
      * since build `302.15.001`
      */
-    displaySet21: types.TUint32,
+    displaySet21: IDisplaySet1OperatorParameter,
```

Comment:
```suggestion
    displaySet21: IDisplaySet21OperatorParameter,
```
?

(and in other fields)